synedrion/src/cggmp21/conversion.rs

```diff
@@ -66,10 +77,10 @@ pub(crate) fn scalar_from_signed<P: SchemeParams>(
 }
 
 /// Converts a secret-wrapped uint to a secret-wrapped [`Scalar`], reducing the value modulo curve order.
-fn secret_scalar_from_uint<P: SchemeParams>(
-    value: &Secret<<P::Paillier as PaillierParams>::Uint>,
+fn secret_scalar_from_wide_uint<P: SchemeParams>(
+    value: &Secret<<P::Paillier as PaillierParams>::WideUint>,
 ) -> Secret<Scalar<P>> {
-    let r = value % &P::CURVE_ORDER;
+    let r = value % &P::CURVE_ORDER_WIDE;
```

Comment:
Maybe it's possible to use `RemMixed` here and get a smaller `r` right away. Probably not worth the hassle though?

Reply:
It's only implemented up to `U1024`. I think it would be much more useful to have access to `rem_vartime()` here, but it doesn't have an associated trait.

Reply:
Filed https://github.com/entropyxyz/synedrion/issues/201

Reply:
> I think it would be much more useful to have access to `rem_vartime()` here, but it doesn't have an associated trait.

`RemMixed` is impl'd with `div_rem_vartime`, which is what `rem_vartime` uses too, so I think we're saying the same thing?

Reply:
> `RemMixed` is impl'd with `div_rem_vartime`

That's actually quite unexpected. I don't think it should use vartime.

Reply:
I named the trait `RemVartime` originally, but that's not a great name either given it operates on mixed input. I guess `RemMixedVartime` would have conveyed the point fully but it's a bit of a mouthful.